example/web-node-sample/README.md

```diff
@@ -0,0 +1,20 @@
+# Web-node validation
+This sample code is to demonstrate that the node sdk can be used to execute Scalar DL binary request build by another application using any Scalar sdk.
+
+![](img/scalar_arch.png)
```

Comment:
I was thinking there could be many other clients (Client A, B, C ...), and same goes for the server A... well i'll use web-client and web-service...

Reply:
```
I was thinking there could be many other clients (Client A, B, C ...), and same goes for the server A
```
It's a valid point that a client can generate a request and any server would be able to execute it but you would have needed to explained it because otherwise it's a concept that is difficult to understand for a developer that is not familiar with Scalar DL. 

I think that by keeping things short and concise in the sample, it will have more impact. That's why I think we can just keep architecture schema which only describe what this sample contains.

Reply:
I also planned to move the index.js file on `example/index.js` into `example/sample/index.js`

Reply:
`example/sample` may be a bit redundant as `sample` as the same meaning as example. I think we can do better
Since the `index.js` has is testing all the methods of the client, what do you think about 
`example/generic` ? I can't really find something better as of now. 
Do you have a better idea ?

Reply:
I'm sorry, I just realized that index.js does not need to have a package.json file by itself, i guess it is fine as the way it is (which is without its own directory)

Reply:
That's fine too